playbooks/pmm2-client_integration_custom_path.yml

```diff
@@ -80,193 +49,61 @@
-    set_fact:
-      random_pgsql_password: "psql-{{ lookup('password', '/tmp/pgsqlpassword chars=ascii_lowercase,digits length=8') }}"
-
-  - name: Change Postgresql Password
-    command: psql -c "ALTER USER postgres PASSWORD '{{ random_pgsql_password }}';"
-    become: yes
-    become_user: postgres
-    when: ansible_distribution_release != 'xenial'
-
-  - name: add postgresql metrics to monitoring
-    command: pmm-admin add postgresql --password={{ random_pgsql_password }} --service-name=pgsql_{{ instance_ip }}
-    when: ansible_distribution_release != 'xenial'
-
-  - name: check if postgresql monitoring is running
-    command: pmm-admin list
-    register: list_output
-    when: ansible_distribution_release != 'xenial'
-
-  - name: Print the pmm-admin list value
-    ansible.builtin.debug:
-      msg: System {{ list_output.stdout }}
-    when: ansible_distribution_release != 'xenial'
-
-  - name: Assert if the pmm-admin list contains postgreSQL instance
-    assert:
-      that:
-        - "'PostgreSQL' in list_output.stdout"
-    when: ansible_distribution_release != 'xenial'
-
-  - name: Wait for postgres_exporter to have Running Status.
-    shell: pmm-admin list | grep 'postgres_exporter' | awk -F" " '{print $2}'
-    when: ansible_distribution_release != 'xenial'
-    register: postgres_exporter_status
-    until: postgres_exporter_status.stdout.find('Running') != -1
-    delay: 5
-    retries: 5
-
-  - name: Assert if the pmm-admin list contains postgres_exporter with Running state
-    assert:
-      that:
-        - "'Running' in postgres_exporter_status.stdout"
-    when: ansible_distribution_release != 'xenial'
+### start pgsql 13
+  - name: install and start Percona Server MongoDB 4.4
+    include_tasks: ../tasks/start_psql13.yml
+
+# there was no metric check and removing - add it?
+
+#  - name: check if Metric End point with Auth works for Postgres Exporter
+#    shell: "{{item}}"
+#    with_items:
+#      - bash -x ../scripts/pmm2_check_metric.sh --service_name pgsql_{{ instance_ip }} --metric_name pg_up{collector=\"\custom_query.hr\"\} --pmm_server_ip {{pmm_server_address}} --agent_type postgres_exporter --metric_value 1
+#      - bash -x ../scripts/pmm2_check_metric.sh --service_name pgsql_{{ instance_ip }} --metric_name pg_up{collector=\"\custom_query.lr\"\} --pmm_server_ip {{pmm_server_address}} --agent_type postgres_exporter --metric_value 1
+#      - bash -x ../scripts/pmm2_check_metric.sh --service_name pgsql_{{ instance_ip }} --metric_name pg_up{collector=\"\custom_query.mr\"\} --pmm_server_ip {{pmm_server_address}} --agent_type postgres_exporter --metric_value 1
+#      - bash -x ../scripts/pmm2_check_metric.sh --service_name pgsql_{{ instance_ip }} --metric_name pg_up{collector=\"\exporter\"\} --pmm_server_ip {{pmm_server_address}} --agent_type postgres_exporter --metric_value 1
+#
+#  - name: check if Metric End point with Auth works for Postgres Exporter when service added via socket
```

Comment:
any reason these are commented?

Reply:
Yes, these check were originally absent in this test. The has been extracted from refactored task. 
so I've added it comented with the question at line 85
`# there was no metric check and removing - add it?`